scripts/platforms/greenhouse_apply_v2.py

```
@@ -0,0 +1,550 @@
+                if any(token in combined for token in ["terms", "privacy", "consent", "certify", "acknowledge"]):
+                    await checkbox.check()
+                    field_memory[combined] = "checked"
+
+            except Exception:
+                continue
+
+        # ====== FINISH ======
+        save_company_memory(company_name, field_memory)
+        print(f"\n✓ Memory saved for {company_name}")
+        print("Review the form before submitting.")
+        input("Press ENTER after submitting manually...")
```

Ruff (0.15.12)

[warning] Blocking call to `input()` in async context

(ASYNC250)